openedx/adg/lms/applications/management/commands/tests/test_update_is_bu_prerequisite_courses_passed.py

```diff
@@ -0,0 +1,125 @@
+"""
+Tests to check if `update_is_bu_prerequisite_courses_passed` command actually updates the
+`is_bu_prerequisite_courses_passed` flag in Application Hub for each eligible user
+"""
+import logging
+
+import pytest
+from django.core.management import call_command
+
+from common.djangoapps.student.tests.factories import CourseEnrollmentFactory, UserFactory
+from lms.djangoapps.grades.models import PersistentCourseGrade
+from openedx.adg.lms.applications.management.commands import update_is_bu_prerequisite_courses_passed as command_module
+from openedx.adg.lms.applications.tests.factories import (
+    ApplicationHubFactory,
+    MultilingualCourseFactory,
+    MultilingualCourseGroupFactory,
+    UserApplicationFactory
+)
+
+
+@pytest.mark.django_db
+def test_command_update_is_bu_prerequisite_courses_passed_with_no_prerequisites():
+    """
+    Test to check if management command issues a system exits upon no prerequisites
+    """
+    with pytest.raises(SystemExit):
+        call_command('update_is_bu_prerequisite_courses_passed')
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('prereq_course_groups', [(1, 'business_line_pre_req')], indirect=True)
```

Comment:
As we are writing `business_line_pre_req` and `program_pre_req` multiple times in the tests, let's make constants for them and use those constants here.